### PR TITLE
testing/php7-pecl-vips: new aport

### PR DESCRIPTION
**Pull request by TBK**
Ping @andypost please take a look.

### CODE REVIEW

File: testing/php7-pecl-vips/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+url="https://pecl.php.net/package/vips"
+arch="all"
+license="MIT"
+depends="php7 vips"
+makedepends="php7-dev autoconf vips-dev"
+source="$pkgname-$pkgver.tgz::https://pecl.php.net/get/$_pkgname-$pkgver.tgz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+        cd "$builddir"
```
Comment (andypost) [inline testing/php7-pecl-vips/APKBUILD]: that command no longer needed, the same for `check()`

File: testing/php7-pecl-vips/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=php7-pecl-vips
+_pkgname=vips
+pkgver=1.0.9
+pkgrel=0
+pkgdesc="PHP extension for interfacing with vips - PECL"
+url="https://pecl.php.net/package/vips"
+arch="all"
+license="MIT"
+depends="php7 vips"
```
Comment (andypost) [inline testing/php7-pecl-vips/APKBUILD]: I bet it should be just `php7-common` because vips will be added by builders

**Comment by TBK** [comment]
@andypost I have made the changes.

**Comment by andypost** [comment]
https://build.alpinelinux.org/buildlogs/build-edge-s390x/testing/php7-pecl-vips/php7-pecl-vips-1.0.9-r0.log
vips is missing for s390x

**Comment by TBK** [comment]
I will make a PR ASAP.

**Comment by andypost** [comment]
@TBK also please add following
```diff
-depends="php7-common vips"
+depends="php7-common"
```

**Comment by TBK** [comment]
Damn I missed that :/ I will make a PR to amend it.

**Comment by TBK** [comment]
Ahh sneaky, so you already fixed it as part of #6321 @andypost thank you 👍

**Pull request closed without merging**